api-tester/src/main/java/com/revenuecat/apitester/java/EntitlementInfoAPI.java

```diff
@@ -72,6 +72,7 @@ static void store(final Store store) {
             case RC_BILLING:
             case EXTERNAL:
             case PADDLE:
+            case TEST_STORE:
```

Comment:
Added this since I think it makes sense to differentiate on the store... If/when we want to merge this, to avoid the extra API, I can temporarily remove it. Same for the `PurchaseType` enum.